db/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import Column, Integer, String
+
+
+class Base(DeclarativeBase): pass
+
+
+class Person(Base):
```

Comment:
Класс модеи таблицы и таблица должны быть одинаково названы. это бест практис
```suggestion
class User(Base):
```